hipchat.go

```diff
@@ -130,10 +134,21 @@ func (c *Client) PostMessage(req MessageRequest) error {
 		return err
 	}
 
-	resp, err := http.PostForm(uri, payload)
+	reqs, err := http.NewRequest("POST", uri, bytes.NewBufferString(payload.Encode()))
```

Comment:
instead of `bytes.NewBufferString(payload.Encode())`, use `strings.NewReader(payload.Encode())`